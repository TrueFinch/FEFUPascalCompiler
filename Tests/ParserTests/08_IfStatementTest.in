program a;
var
    i: integer;
begin
    i := 1;
    if (i > 0) then
        pass
    else
        pass;
end.