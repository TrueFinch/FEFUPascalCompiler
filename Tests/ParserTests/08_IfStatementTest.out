                   a                    
                   ┼                    
               MainBlock                
       ┌───────────┴──────┐             
      var         CompoundStatement     
       ┼         ┌────────┴──┐          
 SimpleVarDecl   :=     IfStatement     
 ┌─────┼        ┌┴─┐   ┌─────┴─────┐    
 i  integer     i  1   >    pass  pass  
                      ┌┴─┐              
                      i  0              